server/asset/doge/doge.go

```diff
@@ -49,7 +49,7 @@ const (
 	version   = 0
 	BipID     = 3
 	assetName = "doge"
-	feeConfs  = 10
+	feeConfs  = 8
```

Comment:
https://github.com/decred/dcrdex/pull/1558#discussion_r849436237

The table I pasted there is still accurate even today.